src/archBuild_sconscript

```diff
@@ -33,11 +33,21 @@ sources = [
 ]
 
 if env["PLATFORM"] == "win32":
-	env.Append(CXXFLAGS="/EHsc /std:c++20 /W3 /WX "
-		# #479: fmt/format.h includes UTF-8 characters in comments. However, cl
-		# treats the file as ANSI unless it has a BOM. This causes a warning (and
-		# thus an error) on systems with certain locales. Just disable this warning.
-		"/wd4819")
+	# On Windows, OSARA is build with LLVM to have a toolchain that's closer to what's used on Mac
+	env["CC"] = "clang-cl"
+	env["LINK"] = "lld-link"
+	env.Append(CXXFLAGS=[
+		"/clang:-std=c++20",
+		"/EHsc",
+		"/W3",
+		"/WX",
+		# Ignore some warnings (treated as errors)
+		# Clang warns about unused functions in WDL/wdltypes.h
+		"/clang:-Wno-unused-function",
```

Comment:
I don't love that we're disabling this for the entire project due to WDL. I think we might want to restrict this by doing this with a pragma at include time in future, but we can deal with that later.